Source/Assets/Scripts/PlayerInteractor.cs

```diff
@@ -0,0 +1,132 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using TMPro;
+using UnityEngine;
+
+public class PlayerInteractor : MonoBehaviour
+{ 
+    
+    private static PlayerInteractor _instance;
```

Comment:
Rework to use reference fetched in Awake() using either FindObjectByType or assigned in the inspector window.